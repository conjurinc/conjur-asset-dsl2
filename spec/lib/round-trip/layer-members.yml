- !members
  role: !layer-use dev/mysql
  members:
    - !group developers

- !members
  role: !layer-admin dev/mysql
  members:
    - !group ops
