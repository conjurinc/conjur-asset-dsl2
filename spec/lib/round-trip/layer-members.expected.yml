---
- !members
  role: !layer-use
    id: dev/mysql
  member:
  - !member
    role: !group
      id: developers
- !members
  role: !layer-admin
    id: dev/mysql
  member:
  - !member
    role: !group
      id: ops
